test/test_smbo/test_acquisition.py

```diff
@@ -3,7 +3,15 @@
 
 import numpy as np
 
-from smac.optimizer.acquisition import EI, LogEI, EIPS, PI, LCB, IntegratedAcquisitionFunction
+from smac.optimizer.acquisition import (
```

Comment:
Are the parentheses necessary?
I don't like this ...

Reply:
Not strictly, but there are two reasons for this coding style:

1. Simpler diffs for future changes: https://stackoverflow.com/a/14377271
2. Suggested by PEP328: https://www.python.org/dev/peps/pep-0328/

Reply:
hmpf